.github/workflows/checks.yml

```diff
@@ -0,0 +1,11 @@
+name: checks
+run-name: Spot check for errors
+on:
+  - push
+jobs:
+  checks:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: cachix/install-nix-action@v20
+      - run: scripts/ci_check.sh
```

Comment:
Eventually, I hope we can change this to `nix flake check` where it checks the flake being good and runs your check script, but that can happen later.

Reply:
:+1: